src/components/CircularProgress/CircularProgress.js

```diff
@@ -9,17 +9,28 @@ import PropTypes from 'prop-types';
  * import { CircularProgress } from 'weaveworks-ui-components'
  *
  * React.render(
- *   <CircularProgress size={45} />
+ *   <div>
+ *     <CircularProgress size={35} />
+ *     <CircularProgress inline /> Inline
+ *     <CircularProgress center />
+ *   </div>
  * );
  * ```
  */
 class CircularProgress extends React.Component {
   render() {
-    const {size} = this.props;
+    const {size, inline, center} = this.props;
     const styles = {
       width: size,
       height: size
     };
+    if (inline) {
+      styles.display = 'inline-block';
+      styles.verticalAlign = 'middle';
+    }
+    if (center) {
+      styles.margin = '0 auto';
+    }
 
     return (
       <div className="weave-circular-progress-wrapper" style={styles}>
```

Comment:
I think it would be preferable to do this by creating a new `CircularProgress` styled component and then forwarding `inline` & `center` to it as props. We're generally trying to move towards styled components whenever possible and `ui-components` repo is a good example where it would work nicely. Maybe you can even remove the `className` here and then update the styles in the PR that bumps the version in `service-ui`.

Reply:
Agreed, but I'd rather this to be done in a separate PR.